index.js

```diff
@@ -65,7 +65,14 @@ BroccoliMergeTrees.prototype.build = function() {
 
   var applyPatchStart = new Date();
 
-  this._applyPatch(patch);
+  try {
+    this._applyPatch(patch);
+  } catch(e) {
+    // Whatever the failure, start again and do a complete build next time
+    this._currentTree = FSTree.fromPaths([]);
```

Comment:
An alliterative, is to scan the output to figure out the actual state. But i suspect just bailing and trying again is  likely just as good.